src/core/default/commands/relationaldb/shell.py

```diff
@@ -23,15 +22,67 @@ def add_arguments(self, parser: ArgumentParser) -> None:
             type=str,
             help="The database to execute on. Name must include component name. ex: comp1.myDb",
         )
+        parser.add_argument("--c", nargs="+", type=str, help="sql command to execute")
+        parser.add_argument("--command", nargs="+", type=str, help="sql command to execute")
+        parser.add_argument("--f", nargs="+", help="execute sql commands from a file")
 
     def command(self, *args, **kwargs) -> None:
 
-        component_name, database_name = self.get_component_and_resource_from_qualified_name(kwargs.get("resource"))
+        (
+            component_name,
+            database_name,
+        ) = self.get_component_and_resource_from_qualified_name(kwargs.get("resource"))
+        c_command = kwargs.get("c")
+        f_command = kwargs.get("f")
         cluster_arn, secret_arn, db_name = get_db_info_from_cdev_name(
             component_name, database_name
         )
-
-        interactive_shell(fmt(Console()), cluster_arn, secret_arn, db_name).cmdloop()
+        if c_command is not None:
+            c_command = self.concatenate_arg_command(c_command)
```

Comment:
Since we are able to now preserve the sql into a single string we do not need this method. `c_command` is a list so you do need to get the actual string out of the list.